jmclient/jmclient/configure.py

```diff
@@ -392,7 +394,7 @@ def load_program_config(config_path=None, bs=None):
         global_singleton.config.set("BLOCKCHAIN", "blockchain_source", bs)
     # Create default config file if not found
     if len(loadedFiles) != 1:
-        with open(global_singleton.config_location, "w") as configfile:
+        with open(global_singleton.config_location, "wb") as configfile:
```

Comment:
Why 'wb', if it's a text file?

Reply:
defaultconfig is bytes which is why I'm writing as bytes

Reply:
Actually it's not (right now). I recall it was at one point.